src/_common/comment/widget/comment/comment.vue

```diff
@@ -23,7 +23,7 @@
 		</template>
 
 		<template #meta>
-			<app-popper v-if="user">
+			<app-popper v-if="user" popover-class="list-group-dark">
```

Comment:
Can we instead just use the correct "fill-X" class? We have the list-group-dark in the popover slot below, so I think we just need this to change the BG to the correct color. I'm guessing list-group-dark is the same as fill-dark, but I don't know.

Reply:
And then be sure to update all the places doing the same thing.